contracts/src/IRiscZeroVerifier.sol

```diff
@@ -49,26 +51,23 @@ struct ReceiptClaim {
 
 library ReceiptClaimLib {
     using OutputLib for Output;
+    using SystemStateLib for SystemState;
 
     bytes32 constant TAG_DIGEST = sha256("risc0.ReceiptClaim");
 
-    /// @notice Construct a ReceiptClaim from the given imageId, postStateDigest, and journalDigest.
+    /// @notice Construct a ReceiptClaim from the given imageId and journalDigest.
     ///         Returned ReceiptClaim will represent a successful execution of the zkVM, running
     ///         the program committed by imageId and resulting in the journal specified by
     ///         journalDigest.
-    /// @dev Input hash is set to all-zeros (i.e. no committed input), the exit code is (Halted, 0),
-    ///      and there are no assumptions (i.e. the receipt is unconditional).
     /// @param imageId The identifier for the guest program.
-    /// @param postStateDigest A hash of the final memory state.
     /// @param journalDigest The SHA-256 digest of the journal bytes.
-    function from(bytes32 imageId, bytes32 postStateDigest, bytes32 journalDigest)
-        internal
-        pure
-        returns (ReceiptClaim memory)
-    {
+    /// @dev Input hash and postStateDigest are set to all-zeros (i.e. no committed input, or
+    ///      final memory state), the exit code is (Halted, 0), and there are no assumptions
+    ///      (i.e. the receipt is unconditional).
+    function from(bytes32 imageId, bytes32 journalDigest) internal pure returns (ReceiptClaim memory) {
         return ReceiptClaim(
             imageId,
-            postStateDigest,
+            SystemState(0, bytes32(0)).digest(),
```

Comment:
I wonder wouldn’t a constant be more efficient here?

Reply:
Good idea. I probably shouldn't trust the Solidity compiler to automatically turn that into a constant.

Reply:
Addressed in https://github.com/risc0/risc0-ethereum/pull/105/commits/b5d9e7dd0fd068aebcc727ff5a3f13b7555073ea